packages/EasyStandard/src/Rector/ExplicitBoolCompareRector.php

```diff
@@ -67,50 +68,46 @@ public function getNodeTypes(): array
     }
 
     /**
-     * {@inheritdoc}
+     * @param If_|ElseIf_ $node
      */
     public function refactor(Node $node): ?Node
     {
-        /** @var \PhpParser\Node\Stmt\If_|\PhpParser\Node\Stmt\ElseIf_ $ifNode */
-        $ifNode = $node;
```

Comment:
Rector tip: it's better to put type in the `@param`, then always check it here. The `@param` is expected location to find type in. And IDE understands it very well